core/backend/src/test/standalone/ExportGraphics.test.ts

```diff
@@ -1176,4 +1179,56 @@ describe("exportGraphics", () => {
     assert.strictEqual(infos[0].mesh.indices.length, 6);
   });
 
+  it("verifies subset visitor", () => {
```

Comment:
How about `VerifySubsetVisitor` to keep test name consistency

Reply:
I used the test naming convention prevalent in this file.